### PR TITLE
chore: add Sergio and Vladimir as code owners

### LINKED ISSUE #844: Sergio joining as code owner
Hey fellow code owners @derberg, @dalelane, and @asyncapi-bot-eve 😄 

I'd like to propose that @smoya joins us as a code owner of the `spec` and `spec-json-schemas` repo. I think it's obvious how much he's been contributing lately and he deserves it IMHO. @smoya I hope you also want to accept this (questionable) honor :)

Leave your comments below!

### LINKED ISSUE #845: Vladimir joining as code owner
Hey fellow code owners @derberg and @dalelane,

I'd like to propose that @char0n joins us as a code owner of the `spec` and `spec-json-schemas` repo. I think it's obvious how much he's been contributing lately and he deserves it IMHO. @char0n I hope you also want to accept this (questionable) honor :)

Leave your comments below!

### PR DESCRIPTION
**Pull request by fmvilas**
### Related issues

Closes #844 
Closes #845 

https://github.com/asyncapi/spec-json-schemas/pull/271

### CODE REVIEW

**Comment by derberg** [comment]
@fmvilas please 
- update description in the way that related issues will be closed after merge
- open the same PR in `spec-json-schema` repo
- I'd like to suggest that with the merge we also agree on some official written-down review rules. For now, we had an unwritten rule that all codeowners had to approve PRs, but we did not have it defined in the settings of branch protection. Imho, we should stay with 3 minimum approvers + some time for others to have time to vote. Cause getting 5 approvers will be super hard for each PR without any deadlines.

File: CONTRIBUTING.md
Comment (smoya) [inline CONTRIBUTING.md:244]: @fmvilas When reading this, I understood those two bullets should be interpreted as `OR` and not as `AND`, meaning one and only one of those should happen in order to merge it.

It's not written wrongly, but I wonder if someone else would understand it in the same way I did on my first read, so I would like to suggest we clarify both points should be fulfilled.

Comment (fmvilas) [reply CONTRIBUTING.md:244]: That's a good point. Fixed it, I think :)

**Review by smoya — APPROVED** [review]
❤️

**Comment by fmvilas** [comment]
Alright, can I get some ✅ here? 😄 @dalelane @derberg

**Review by derberg — APPROVED** [review]
lgtm

@char0n have a look so you are also aware of changes and agree

**Review by dalelane — APPROVED** [review]
🎉

**Review by char0n — APPROVED** [review]
LGTM

**Comment by derberg** [comment]
@char0n As a maintainer in one of AsyncAPI projects (`spec` and `spec-json-schema`) you also have the right to join the AsyncAPI TSC and participate in the decision-making process for the entire AsyncAPI Initiative. Just lemme know if you are interested to learn more.

**Comment by char0n** [comment]
@derberg sure I'm always interested to learn more ;]

**Comment by derberg** [comment]
@char0n awesome, please open a PR like Jerremy did recently https://github.com/asyncapi/community/pull/473 and mention all the 3 repos where you are now a codeowner please

**Pull request merged**
